doc/tutorial/Authentication.lhs

```diff
@@ -281,17 +281,22 @@ lookupAccount key = case Map.lookup key database of
 For generalized authentication, servant exposes the `AuthHandler` type,
 which is used to wrap the `Request -> Handler user` logic. Let's
 create a value of type `AuthHandler Request Account` using the above `lookupAccount`
-method:
+method (note: we depend upon `cookie`'s `parseCookies` for this):
```

Comment:
Pushed _something_ - is there a way to see what it actually looks like? Not a fan of the guessing game! Looks ok in general markdown formatters.